### ISSUE TITLE
License of the dataset and code

### ISSUE THREAD

**Issue by jnhwkim** [skokec/segdec-net-jim2019#25]
Could you specify what kind of license do you permit to use the code and the dataset?

**Comment by skokec** [comment]
Hi, both the code and the dataset are available for research and non-commercial use only.

**Comment by jnhwkim** [comment]
Presumably, the same license for KolektorSDD2?

**Comment by skokec** [comment]
Yes, KolektorSDD2 is also available for research and non-commercial use only.
